actors/common/src/main/java/org/sunbird/feed/FeedUtil.java

```diff
@@ -60,11 +72,25 @@ private static Feed createFeedObj(ShadowUser shadowUser, String userId) {
     List<String> channelList = new ArrayList<>();
     channelList.add(shadowUser.getChannel());
     prospectsChannel.put(JsonKey.PROSPECT_CHANNELS, channelList);
+    prospectsChannel.put(JsonKey.PROSPECT_CHANNELS_IDS, getOrgDetails(shadowUser.getOrgExtId()));
     feed.setData(prospectsChannel);
     feed.setUserId(userId);
     return feed;
   }
 
+  private static List<Map<String, String>> getOrgDetails(String orgExtId) {
+    Map<String, Object> filters = new HashMap<>();
+    filters.put(JsonKey.EXTERNAL_ID, orgExtId);
+    Organisation org = organisationClient.esSearchOrgByFilter(filters).get(0);
```

Comment:
Since orgids and orgnames are pretty much constant, can you please add a simple hashmap of those - we can reduce calls to ES that is already loaded for the high reads presently?